components/stock-operation-modal/index.js

```diff
@@ -17,8 +17,8 @@ const StockOperationModal = (props) => {
   } = props;
 
   const [quantity, setQuantity] = useState('');
-  const [isUserLoggedIn, setIsUserLoggedIn] = useState('');
-  const [userMoney, setUserMoney] = useState();
+  const [isUserLoggedIn, setIsUserLoggedIn] = useState('false');
```

Comment:
Shouldn't this be boolean?
`useState(false)`

Reply:
Okay. I will change that 👍